rdflib/extras/infixowl.py

```diff
@@ -131,63 +130,56 @@
 """
 
 __all__ = [
-    "nsBinds",
     "ACE_NS",
-    "CLASS_RELATIONS",
-    "some",
-    "only",
-    "max",
-    "min",
-    "exactly",
-    "value",
-    "PropertyAbstractSyntax",
     "AllClasses",
     "AllDifferent",
     "AllProperties",
     "AnnotatableTerms",
     "BooleanClass",
+    "CLASS_RELATIONS",
     "Callable",
     "CastClass",
     "Class",
     "ClassNamespaceFactory",
-    "classOrIdentifier",
-    "classOrTerm",
     "CommonNSBindings",
     "ComponentTerms",
     "DeepClassClear",
     "EnumeratedClass",
-    "generateQName",
     "GetIdentifiedClasses",
     "Individual",
     "Infix",
-    "MalformedClass",
-    "manchesterSyntax",
-    "Ontology",
+    "MalformedClassError",
     "OWLRDFListProxy",
+    "Ontology",
     "Property",
-    "propertyOrIdentifier",
+    "PropertyAbstractSyntax",
     "Restriction",
-    "termDeletionDecorator",
+    "classOrIdentifier",
+    "classOrTerm",
+    "exactly",
+    "generateQName",
+    "manchesterSyntax",
+    "max",
+    "min",
+    "nsBinds",
+    "only",
+    "propertyOrIdentifier",
+    "some",
+    "value",
 ]
 
 # definition of an Infix operator class
 # this recipe also works in jython
 # calling sequence for the infix is either:
-#  x |op| y
+#  x << op >> y
 # or:
-# x <<op>> y
+#  x @ op @ y
 
 
 class Infix:
     def __init__(self, function):
         self.function = function
 
-    def __ror__(self, other):
-        return Infix(lambda x, self=self, other=other: self.function(other, x))
-
-    def __or__(self, other):
-        return self.function(other)
-
```

Comment:
Can't we rather mark these as deprecated?

Reply:
It's a bit late for that :smile: https://github.com/RDFLib/rdflib/blob/81b36e2968e18764d30c48295e3087f1836ecbe8/test/test_extras/test_infixowl/test_cover.py#L64